tests/utils/actions/keyboard.ts

```diff
@@ -97,7 +97,11 @@ export async function formatType(page: Page) {
   });
 }
 
-export async function redoByKeyboard(page: Page) {
+export async function redoByKeyboard(page: Page, onlyWin = false) {
+  if (onlyWin && IS_WINDOWS) {
```

Comment:
This is not a valid test, `IS_WINDOWS` should be always false on CI. Given the situation that cross-OS testing suite is not set up, not testing this PR is just fine.